src/test/java/com/google/javascript/clutz/partial/declare_legacy_namespace.js

```diff
@@ -0,0 +1,7 @@
+
+goog.module('declare.legacy.namespace');
+goog.module.declareLegacyNamespace();
+
+class Class {}
+
+exports = Class;
```

Comment:
add a test for named export too, code does have a branch on that.

Reply:
We already have one - goog_legacy_namespace_exporter.d.ts (below) is the result of just such a circumstance.